CHANGELOG.cdf-tk.md

```diff
@@ -28,12 +30,12 @@ Changes are grouped as follows:
 - Added support for running a Cognite function locally using the `run function --local` command. This command will run the
   function locally in a virtual environment simulating CDF hosted run-time environment and print the result to the console.
 
-## Changed
+### Changed
 
-- The cdf-toolkit now requires one `config.yaml` per environment, for example, `config.dev.yaml` and `config.prod.yaml`.
-- The file `environments.yaml` has been merged into `config.[env].yaml`. This means that the `environments.yaml` file
-  is no longer used and the `config.[env].yaml` file now contains all the information needed to deploy to
-  that environment.
+- **BREAKING:** The cdf-toolkit now requires one `config.yaml` per environment, for example, `config.dev.yaml` and `config.prod.yaml`.
+- **BREAKING:** The file `environments.yaml` has been merged into `config.[env].yaml`.
```

Comment:
Maybe add `cdf-tk init [my_project] --upgrade` gives you step-by-step instructions to migrate.

Reply:
Already in there: https://github.com/cognitedata/cdf-project-templates/blob/0376ebfccc08873fb75de62b95332ca59216acba/CHANGELOG.cdf-tk.md?plain=1#L44-L48